ATTRIBUTIONS/wasabi-email-attribution.xml

```diff
@@ -1214,7 +1214,7 @@
          <type>jar</type>
          <licenses/>
          <downloadUrls>
-            <string>TBD</string>
+            <string>file:/Users/jwu/git/wasabi-core/modules/email/../../repository/Hyrule/Hyrule/0.0.1/Hyrule-0.0.1.jar</string>
```

Comment:
yes ... this is an unfortunate artifact of having a project internal mvn repo for hyrule, which we will fix.

till then, run:

`  % ./bin/ATTRIBUTIONS_patch.sh`

then git add, etc.